### PR TITLE
proposed revisions to entity programming model

### PR DESCRIPTION
**Pull request by gavinking**
Thoughts?

### CODE REVIEW

File: spec/src/main/asciidoc/repository.asciidoc [hunk 1]
```diff
@@ -146,59 +146,91 @@ NOTE: Jakarta Data allows applications to intermix both patterns by defining met
...
+NOTE: When there's no risk of confusion, we often use the word “entity” to mean the entity class, or even an instance of the entity class.
 
-3. *Persistence and Representation*: Entities are associated with persistent data, meaning the data outlives any specific Java process utilizing it. Each persistent instantiation of the schema is distinguishable by a unique identifier. For example, a row of a relational database table is identifiable by the value of its primary key. In Java, these entities are represented as classes, referred to as entity classes. It's important to note that multiple instances of the entity class within a Java program can represent a single persistent instance of the schema.
+Data represented by an entity is persistent, that is, the data itself outlives any Java process which makes use of it. Thus, it's necessary to maintain an association between instances of Java entity classes, and state held in a data store.
```
Comment (njr-11) [inline spec/src/main/asciidoc/repository.asciidoc:158]: Minor grammar correction to remove a comma,

```suggestion
Data represented by an entity is persistent, that is, the data itself outlives any Java process which makes use of it. Thus, it is necessary to maintain an association between instances of Java entity classes and state held in a data store.
```

**Review by njr-11 — APPROVED** [review]
This looks great.  Nice improvements!   I spotted one minor grammar correction to make removing an extra comma.

File: spec/src/main/asciidoc/repository.asciidoc (hunk 1 above)
Comment (gavinking) [reply spec/src/main/asciidoc/repository.asciidoc]: Thanks, good catch.

**Review by otaviojava — APPROVED** [review]

**Review by njr-11 — APPROVED** [review]
The additional commit looks good as well.

**Pull request merged**